### PR TITLE
Typo "Teams Room"→"Teams Rooms"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/hybrid-meetings-device-config-intro 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/hybrid-meetings-device-config-intro.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/hybrid-meetings-device-config-intro.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The following steps show you how to purchase a license for your Teams Rooms cons
 
 ## Configure mailbox and account properties
 
-To allow people your organization to book the Teams Room, you need to configure a few properties on the account the console uses. Setting these properties allows the Teams Room to process meeting requests automatically the way you want, remove or add information shown about the meeting on the console, and set what should be included in meeting request responses.
+To allow people your organization to book the Teams Room, you need to configure a few properties on the account the console uses. Setting these properties allows the Teams Rooms to process meeting requests automatically the way you want, remove or add information shown about the meeting on the console, and set what should be included in meeting request responses.
```
Comment (JohanFreelancer9) [inline Teams/hybrid-meetings-device-config-intro.md:47]:
```suggestion
To allow people in your organization to book the Teams Room, you need to configure a few properties on the account the console uses. Setting these properties allows Teams Rooms to process meeting requests automatically the way you want, remove or add information shown about the meeting on the console, and set what should be included in meeting request responses.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the suggested change. Many thanks.

File: Teams/hybrid-meetings-device-config-intro.md (hunk 1 above)
Comment (hyoshioka0128) [reply Teams/hybrid-meetings-device-config-intro.md:47]: @JohanFreelancer9 - Thanks for the suggestion.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks for the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**